src/Wallet.ts

```diff
@@ -107,6 +108,21 @@ class Wallet {
     const signature = tx.TxnSignature
     return verify(messageHex, signature, this.publicKey)
   }
+
+  /**
+   * Gets an X-address in testnet/mainnet format.
+   * @param {number} tag A tag to be included within the XAddress.
+   * @param {boolean} test A boolean to indicate if XAddress should be in testnet (true) or mainnet (false) format.
+   * @returns {string} An X-address
+   */
+  getXAddress(tag: number, test: boolean): string {
+    const classicAddress = deriveAddress(this.publicKey)
+    return classicAddressToXAddress(
+      classicAddress,
+      tag,
+      test,
+    )
```

Comment:
You can make this slightly more concise:
```suggestion
    return classicAddressToXAddress(
      deriveAddress(this.publicKey),
      tag,
      test,
    )
```